prodtests/full-system-test/workflow-setup.sh

```diff
@@ -8,7 +8,7 @@ SOURCE_GUARD_WORKFLOW_SETUP=1
 #Some additional settings used in this workflow
 
 if [[ $SYNCMODE == 1 ]]; then
-  if [[ -z "${WORKFLOW_DETECTORS_MATCHING+x}" ]]; then export WORKFLOW_DETECTORS_MATCHING="ITSTPC,ITSTPCTRD,ITSTPCTOF,ITSTPCTRDTOF,PRIMVTX"; fi # Select matchings that are enabled in sync mode
+  if [[ -z "${WORKFLOW_DETECTORS_MATCHING+x}" ]]; then export WORKFLOW_DETECTORS_MATCHING="ITSTPC,ITSTPCTRD,ITSTPCTOF,ITSTPCTRDTOF,TPCTRD,TPCTRDTOF,TPCTOF,PRIMVTX"; fi # Select matchings that are enabled in sync mode
```

Comment:
Here I agree with Ruben. Do we really want unconditional matching enabled. Before we do so, we should test in a synthetic run with full rate whether this works. The WORKFLOW_DETECTORS_MATCHING can be overridden in the extra env variables for this.
For reference, there is also no unconstrained TPCTRD matching enabled in sync mode, only in async.

Reply:
I just keep TPC in track sources since it is needed by QC